build_modules/pubspec.yaml

```diff
@@ -33,3 +34,7 @@ dev_dependencies:
     path: test/fixtures/a
   b:
     path: test/fixtures/b
+
+dependency_overrides:
```

Comment:
cruft?

Reply:
Necessary because there is a `dev_dependency` on `build_vm_compilers` which is presently incompatible with this new version bump.